apps/core-app/src/utils/dataFetchHelpers.ts

```diff
@@ -80,16 +80,16 @@ export const loadUserMembership = async ({
     });
 
     if (memberRes?.data?.member && shouldUpdate) {
-      setUserMembership(memberRes.data.member);
+      setMember(memberRes.data.member);
     } else {
-      setUserMembership(undefined);
+      setMember(undefined);
     }
   } catch (error) {
     console.error(error);
```

Comment:
I guess should update goes here too. I'll need to start adding those to my catch statements as well